nims-notes-things/src/components/notes.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styled from "styled-components";
+import { useSelector } from "react-redux";
+
+function Notes() {
+  const notes = useSelector((state) => {
+    return state.noteReducer.value;
+  });
+
+  return (
+    <div>
+      {notes.map((n) => {
+        return (
+          <div>
+            <Note>
+              <pa>{n.name}</pa>
```

Comment:
`<pa />`?

Reply:
that was just me playing around with style component - i forgot to remove it